### ISSUE TITLE
Create a "simple" example in the README

### ISSUE THREAD

**Issue by nitrocode** [cloudposse/terraform-null-label#146]
Have a question? Please checkout our [Slack Community](https://slack.cloudposse.com) or visit our [Slack Archive](https://archive.sweetops.com/). 

[![Slack Community](https://slack.cloudposse.com/badge.svg)](https://slack.cloudposse.com)

## Describe the Feature

The README has a Wall of Text which is overwhelming even for people who are familiar with the null label.

This wall of text prevents people from deliberately adopting (instead of accidentally adopting through usage of the open source modules) this module.

## Expected Behavior

Real simple

```hcl
module "this" {
  namespace = "org"

  environment = "ue1"

  stage = "prod"

  name = "titan"
}
```

`module.this.id` equals `org-ue1-prod-titan`

## Use Case

More adoption, easier to explain

## Describe Ideal Solution

Simplify the readme

## Alternatives Considered

A separate post

## Additional Context

N/A

**Comment by natemccurdy** [comment]
Another thing that might help adoption: explaining, with examples, what problem this module is meant to solve.

I recently found this module because it was used by other CloudPosse modules. As I read the readme, I realized that I didn't really understand why I'd want to use this module. And the wall of text you mentioned further muddied the waters... so I stopped reading.

Are there any real-world examples of folks using the module to go from `<example_of_poorly_named_resources>` to `<example_of_well_named_resources>`?
People might not even realize they're poorly naming things until they see an example of code that they recognize as their own.


Anyway, just my 2 cents as someone recently discovering this module and wondering if I should use it. Thanks for making these modules 🍻

**Comment by nitrocode** [comment]
@natemccurdy I completely agree. I added some use-cases above after your comment that might help explain it a little more.

**Comment by nitrocode** [comment]
cc: @Gowiem posted this https://masterpoint.io/updates/terraform-null-label/
